exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxy.java

```diff
@@ -285,15 +285,15 @@ private UncheckedMapProof getMultiKeyProof(Collection<? extends K> keys) {
   private native UncheckedMapProof nativeGetMultiProof(long nativeHandle, byte[] keys);
 
   /**
-   * Returns the root hash of the underlying Merkle-Patricia tree.
+   * Returns the index hash of this map.
    *
    * @throws IllegalStateException  if this map is not valid
    */
-  public HashCode getRootHash() {
-    return HashCode.fromBytes(nativeGetRootHash(getNativeHandle()));
+  public HashCode getIndexHash() {
```

Comment:
On top of 'object hash' and 'index/collection hash', we might name the thing 'state hash' — we already use the term in Service/Schema APIs.